pvacseq/lib/config_files.py

```diff
@@ -0,0 +1,23 @@
+import sys
+import argparse
+
+def additional_input_file_list_options():
+    return {
+        'gene_expn_file'      : 'genes.fpkm_tracking file from Cufflinks',
+        'transcript_expn_file': 'isoforms.fpkm_tracking file from Cufflinks',
+    }
+
+def main(args_input = sys.argv[1:]):
+    parser = argparse.ArgumentParser('pvacseq config_files')
+    parser.add_argument("config_file_type",
+                        choices=['additional_input_file_list'],
```

Comment:
Yes! We are planning on moving all the optional parameters into a separate config file soon.